scripts/build.sh

```diff
@@ -0,0 +1,30 @@
+#!/usr/bin/env sh
+
+# abort on errors
+set -e
+
+node -v
+
+# build
+pnpm run build
+
+# navigate into the build output directory
+cd dist
+
+# add .nojekyll to bypass GitHub Page's default behavior
+touch .nojekyll
+
+# if you are deploying to a custom domain
+echo 'theguardian.engineering' > CNAME
+
+git init -b main
+git add -A
+git commit -m 'deploy'
+
```

Comment:
Those files already exist so you should not need to do that

Reply:
That’s a quirk of `gh-pages`, you need to publish to that branch, but we only want the content of `dist`, no need to publish the source files themselves.

Reply:
I see

Reply:
But I think it’s best to just use a tried-and-tested GHA https://github.com/marketplace/actions/github-pages-action

Reply:
You might be interested in [this new feature](https://github.blog/changelog/2021-12-16-github-pages-using-github-actions-for-builds-and-deployments-for-public-repositories/).